packages/bundle-source/src/index.js

```diff
@@ -276,15 +276,22 @@ async function bundleZipBase64(startFilename, options = {}, powers = {}) {
       sourceMapJobs.add(writeSourceMap(sourceMap, sourceDescriptor));
     },
   });
+  assert(sha512);
   await Promise.all(sourceMapJobs);
   const endoZipBase64 = encodeBase64(bytes);
   return harden({
-    moduleFormat: 'endoZipBase64',
+    moduleFormat: /** @type {const} */ ('endoZipBase64'),
     endoZipBase64,
     endoZipBase64Sha512: sha512,
   });
 }
 
+/**
+ * @template {'nestedEvaluate' | 'getExport'} T
+ * @param {string} startFilename
+ * @param {T} moduleFormat
+ * @param {*} powers
```

Comment:
annotating out of scope. out of curiosity: do you know it's destructured in the body instead of params?

Reply:
I want to narrow the return type, otherwise the returned `moduleFormat` doesn't match that of the supplied `moduleFormat`.